release-notes/5.0/api-diff/netcoreapp3.1/.Net/5.0_System.Globalization.md

```diff
@@ -0,0 +1,134 @@
+# System.Globalization
+
+``` diff
+ namespace System.Globalization {
+-    public class CompareInfo : IDeserializationCallback {
++    public sealed class CompareInfo : IDeserializationCallback {
```

Comment:
I don't see this in our breaking changes list. `CompareInfo` has been unsealed and had many virtual members going back to at least Full Framework v2.0

Reply:
I don't believe anyone could have derived from them previously, as they lack public/protected ctors.

Reply:
Here's the PR https://github.com/dotnet/runtime/pull/31761.  cc @GrabYourPitchforks 
> There are no public or protected ctors for these types, so nobody is able to subclass them. There's no reason for them to be unsealed. Furthermore, once they're sealed, codegen won't need to emit a virtual dispatch pattern when calling the APIs.